substra/cli/printers.py

```diff
@@ -167,30 +167,35 @@ def _get_list_fields(self):
     def _get_single_fields(self):
         return (self.key_field, ) + self.single_fields
 
-    def print_download_message(self, item):
+    def print_with_profile(self, profile):
```

Comment:
In this module, all methods starting with `print_` actually print something. You could rename it to `get_profile_arg` to make its purpose crystal clear.

Reply:
That's true, I'll change that! 🙂